controller/src/main/java/org/jboss/as/controller/operations/global/GlobalOperationHandlers.java

```diff
@@ -84,6 +84,7 @@ public static void registerGlobalOperations(ManagementResourceRegistration root,
             root.registerOperationHandler(org.jboss.as.controller.operations.global.ReadAttributeHandler.RESOLVE_DEFINITION,
                     org.jboss.as.controller.operations.global.ReadAttributeHandler.RESOLVE_INSTANCE, true);
             root.registerOperationHandler(ReadAttributeGroupHandler.RESOLVE_DEFINITION, ReadAttributeGroupHandler.RESOLVE_INSTANCE, true);
+            root.registerOperationHandler(GlobalInstallationReportHandler.DEFINITION, GlobalInstallationReportHandler.INSTANCE, false);
```

Comment:
This doesn't belong here. I don't think the op is meant to be registered (i.e. second commit is to disable the file report) and if it should be registered, ServerRootResourceRegistration is where it should happen.

I think some confusion probably comes from GlobalOperationHandlers.AbstractMultiTargetHandler. That class' functionality can be used for non "global" ops; it just hasn't been.. A "global" op means it exists on every resource.